pkg/orchestrator/redact/pod.go

```diff
@@ -53,3 +54,11 @@ func ScrubContainer(c *v1.Container, scrubber *DataScrubber) {
 		c.Args = scrubbedMergedCommand[words:]
 	}
 }
+
+// RemoveLastAppliedConfigurationAnnotation redacts the whole "kubectl.kubernetes.io/last-applied-configuration" annotation. As it may contain duplicate information and secrets.
+func RemoveLastAppliedConfigurationAnnotation(annotations map[string]string) {
+	a := annotations["kubectl.kubernetes.io/last-applied-configuration"]
+	if a != "" {
+		annotations["kubectl.kubernetes.io/last-applied-configuration"] = redactedValue
```

Comment:
We might want to use `-` instead to make clear that we did not redact this because we found sensitive words